app/models/publisher.rb

```diff
@@ -59,12 +59,12 @@ class UpholdAccountState
   # Normalizes attribute before validation and saves into other attribute
   phony_normalize :phone, as: :phone_normalized, default_country_code: "US"
 
-  validates :email, email: { strict_mode: true }, presence: true, unless: -> { pending_email.present? }
-  validates :email, uniqueness: { case_sensitive: false }, allow_nil: true
-  validates :pending_email, email: { strict_mode: true }, presence: true, if: -> { email.blank? }
+  validates :email, email: { strict_mode: true }, presence: true, unless: -> { pending_email.present? || deleted? }
```

Comment:
I'm not a big fan of having a conditionals on validations because it makes it harder for our mental model to know what is being validated and when. 

Were these added because of the `PublisherRemovalJob`? Perhaps we should instead call `update_attribute` in the job because the method skips validation.

Reply:
I'll do so.

Reply:
Actually now that I think about it: If we want to make any changes to these deleted users, the validation would continuously fail. I realize it's messy, but it'll be a "wtf is happening" to the next developer who's trying to make a change on a deleted user. The next dev will then also need to remember to use `update_attribute`. 

We could specify individual function to skip these validations: https://richonrails.com/articles/skipping-validations-in-ruby-on-rails

Thoughts?

Reply:
Hmm, let's keep it the weird logic for now. Perhaps we can find a better way of handling it later